src/processing/states/GA.R

```diff
@@ -0,0 +1,79 @@
+# Processing code for Georgia
+
+# Set-up
+this_state <- 'GA'
+change_path(this_state)
+
+# Read in and combine data
+print(sprintf("[%s] reading in the data", this_state))
+
+d <- lapply(list.files('.'), function(fn) {
+    read_csv(fn, col_names=F, col_types=paste(rep('c',103), collapse=''))
+  }) %>%
+  bind_rows()
+
+# separate date and time
+d <- d %>%
+  separate(X4, c("date","time"), " ")
+
+# Group same stop rows to account for multiple violations
+d <- d %>%
+  group_by(date,time,X9,X11,X13,X29,X30,X31,X33,X39,X48,X49,X52,X53,X56) %>%
+  summarize(
+    dob  = get_unique_value_if_exists(X14),
+    sex  = get_unique_value_if_exists(X17),
+    race = get_unique_value_if_exists(X15),
+    loc  = paste(na.omit(unique(X42)), collapse=','),
```

Comment:
Why is the location in the summarize() and not in the groupBy()? Are there instances in the data where there are multiple, non-NA locations conditioning on all fields in the group_b()? Or is it purely to deal with NAs in the location variable?

Reply:
It's the latter, sometimes the one of the values is filled out but not the others (for otherwise identically looking stops).